typescript/test/test_application_ts/src/server.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import * as fs from 'fs-extra';
+import * as child_process from 'child_process';
+import * as kill from 'tree-kill';
```

Comment:
do we use it?

Reply:
I see - but does regular killing not work? (if you run the server directly without a shell)

Reply:
just tried again, regular child_process() actually works. Thanks